<launch>
  <!-- <rosparam file="$(find livelybot_description)/robot_param/12dof_STM32H730_model_P_Hector_params_livelybot.yaml" command="load" /> -->
  <!-- <rosparam file="/home/sunteng/control_ws/src/livelybot_robot/src/livelybot_description/robot_param/18dof_STM32H730_model_H_Hector_params.yaml" command="load" /> -->
  <include file='$(find livelybot_description)/launch/livelybot_description.launch' />
  <!-- <include file='$(find pai_12dof_v2_0312)/launch/display.launch'/> -->
  <node pkg="yesense_imu" type="yesense_imu_node" name="yesense_imu_node" required="true" output = "screen" >
    <param name="yesense_port"      type="string" value="/dev/ttyACM"/>
    <param name="yesense_baudrate"  type="int"    value="460800"/>
  </node>

  <node pkg="joy" name="joy_node_" type="joy_node" output="screen"/>
  <!-- <node pkg="rqt_reconfigure" name="reconfigure_node" type="rqt_reconfigure" output="screen"/> -->
  <!-- <node pkg="sim2real" name="pd_controller" type="pd_controller" output="screen" required="true" />
  <node pkg="sim2real" name="rl_controller" type="rl_controller" output="screen" required="true" /> -->
  <node pkg="sim2real" name="rl_pd_controller" type="rl_pd_controller" output="screen" >
      <!--param name="config_file" value="$(find sim2real)/config.yaml"/>
      <param name="dynamic_config_file" value="$(find sim2real)/dynamic_config.yaml"/-->
  </node>
  <!-- <arg name="bagfile" default="$(find sim2real)/../sim2real_data_log/sim2real_log.bag"/>
  <arg name="topics" default="/mtr_state  /rbt_state  /mtr_target /rbt_target /pd2rl /rl2pd"/>
  <node name="rosbag_record" pkg="rosbag" type="record" args="-o $(arg bagfile) $(arg topics)"/> -->
</launch>
